server/plugin.go

```diff
@@ -502,24 +458,18 @@ func (p *Plugin) handleAccept(w http.ResponseWriter, r *http.Request) {
 	p.PostBotDM(sender, message)
 }
 
-type completeAPIRequest struct {
-	ID string `json:"id"`
-}
-
 func (p *Plugin) handleComplete(w http.ResponseWriter, r *http.Request) {
 	userID := r.Header.Get("Mattermost-User-ID")
 
-	var completeRequest *completeAPIRequest
-	decoder := json.NewDecoder(r.Body)
-	if err := decoder.Decode(&completeRequest); err != nil {
-		p.API.LogError("Unable to decode JSON err=" + err.Error())
-		p.handleErrorWithCode(w, http.StatusBadRequest, "Unable to decode JSON", err)
+	completeRequest, err := GetCompleteIssuePayloadFromJSON(r.Body)
```

Comment:
Is the API `CompleteIssue` or `CompleteOAuth`? Check the naming of variables everywhere.

Reply:
complete Issue only.